packages/ui-kit-components/src/components/IndexTickers/IndexTickers.test.tsx

```diff
@@ -0,0 +1,81 @@
+import '@testing-library/jest-dom';
+
+import { render, screen } from '@testing-library/react';
+
+import IndexTickers from './IndexTickers';
+
+describe('Render the index tickers component', () => {
```

Comment:
If we want to follow the BBD(Behavior Driven Development), Every test statement should start with (Given, When, And, Then). For example: 
```
  describe('Given a customized the text picker ', () => {
    describe("When have latestPrice of 234.89, changedPrice of 1.35, and percentage of 0.34", () => {
      beforeEach(() => {
        render(
          <IndexTickers
            name="SPA"
            latestPrice={234.89}
            changedPrice={1.35}
            percentage={0.34}
          />,
        );
      })
      test('Then "SPA" in the document text', () => {
        expect(screen.getByText('SPA')).toBeInTheDocument();
      });
      test("And Then price `$234.89` should in the document text", () => {
        expect(screen.getByText('$234.89')).toBeInTheDocument();
      })
      test("And Then text `+1.35` should be in the document text", () => {
        expect(screen.getByText('+1.35')).toBeInTheDocument();
      })
      test("And Then text `+0.34%` should be in the document text", () => {
        expect(screen.getByText('+0.34%')).toBeInTheDocument();
      })
    })
  });
```

Reply:
Maybe it's better to add Yu Qi as the reviewer, she could give you more advice :)

Reply:
Yea sounds good!

Reply:
I see that in a lot of your tests you're testing for specific values, but what these values actually are don't matter.
For this component, you only have 4 props. But as the number increases, you don't really want to name all of them in the test.

Suggestion:
```
Given a text picker component
When render with custom props
Then <some element> should be <specific value>
```

So then instead of checking if the value is in document, you could target the specific tag where the value should be displayed and assert on that.

Reply:
Yes, make sense. I'll try to change it based on your suggestion. Thank you!